test/statistical_testing/analyze_results.py

```diff
@@ -0,0 +1,72 @@
+
+import numpy as np
+import argparse
+from scipy.stats import ttest_ind
+
+def check_error(test_name, test_result_file, expected_result_file, confidence=0.95):
```

Comment:
why do you take a confidence argument if you never use it?

Reply:
I want to use it but I don't know how to determine the z-value from the confidence value. Do you?

Reply:
Suppose our confidence is 2c. Then we want to find the value of x such that the Normal CDF takes value .5+c

The CDF is given
.5[1 + erf(x/sqrt2)]

You can find this x value by any means, for instance Newton or even a dumb binary search.
I can implement this tonight if you’d like.